t/t5601-clone.sh

```diff
@@ -68,6 +68,14 @@ test_expect_success 'clone respects GIT_WORK_TREE' '
 
 '
 
+test_expect_success MINGW,CASE_INSENSITIVE_FS 'core.worktree is not added due to path case' '
+
+	mkdir CAPITALCASE &&
+	lower_path=$( cygpath -w "$(pwd)/CAPITALCASE" | tr A-Z a-z ) &&
+	git clone src "$lower_path" &&
+	test -z $(cd "CAPITALCASE" && git config core.worktree)
```

Comment:
Better: `test -z "$(git -C CAPITALCASE config core.worktree)"`

Reply:
Fixed. Copy&pasted that from t0001.